rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -29,7 +29,8 @@ THE SOFTWARE.
 #include "rocal_api.h"
 #include "image_source_evaluator.h"
 
-#define MAX_ASPECT_RATIO 3.0f
+// Calculated from the largest resize shorter dimension in imagenet validation dataset
+#define MAX_ASPECT_RATIO 6.0f
```

Comment:
how much accuracy change is achieved using this change?

Reply:
Without this change, the rocAL classification trainings are not converging and we are achieving around 75.5x val top1 accuracy